cocos/2d/assembler/label/bmfontUtils.ts

```diff
@@ -143,7 +143,7 @@ export const bmfontUtils = {
             // But this function is not called when just modifying the opacity.
             // So the value of '_comp.node._uiProps.colorDirty' does not change.
             // And _uiProps.colorDirty is synchronized with renderEntity.colorDirty.
-            if (_comp.renderEntity.colorDirty || isResized) {
+            if (JSB && (_comp.renderEntity.colorDirty || isResized)) {
```

Comment:
Why need it?

Reply:
Because updateColor only has a native platform implementation